### PR TITLE
fix: fix empty space formatting issue in the rsync job helm template

### PR DESCRIPTION
**Pull request by alex-vmw**
Signed-off-by: Alex Romanenko <alex@vmware.com

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: integration/integration_test.go [hunk 1]
```diff
@@ -52,6 +52,8 @@ const (
 	migrateCmdlineWithNetpols = migrateCmdline +
 		"--helm-set rsync.networkPolicy.enabled=true " +
 		"--helm-set sshd.networkPolicy.enabled=true"
+        migrateCmdlineWithNetpolsAndRsyncFixPrivateKeyPerms = migrateCmdlineWithNetpols +
+                "--helm-set rsync.fixPrivateKeyPerms=true"
```
Comment (utkuozdemir) [inline integration/integration_test.go:56]:
```suggestion
                " --helm-set rsync.fixPrivateKeyPerms=true"
```

**Comment by alex-vmw** [comment]
@utkuozdemir If you could spin up release 1.5.1 whenever you have a chance, would really appreciate it. Thank you in advance.

**Comment by utkuozdemir** [comment]
releasing v1.6.0 now

**Comment by utkuozdemir** [comment]
doing v1.6.1 with correctly including the changes on the sshd image.

**Pull request merged**